pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/filter/participant/BaseFilterParticipantList.java

```diff
@@ -46,6 +50,11 @@ public BaseFilterParticipantList(String jsonBody) {
     @Override
     public ParticipantWrapperResult filter(QueryParamsMap queryParamsMap, Deserializer deserializer) {
         this.deserializer = deserializer;
+
+        //no need to check if realm and userID is set because it was already checked in FilterRoute
```

Comment:
but this is a public method -- how do we know FilterRoute was already called?

Reply:
FilterRoute is entry to the BaseFilterParticipantList

Reply:
other class who is calling it would be `DownloadParticipantListRoute` and this one is also checking the permissions